src/js/Helper/compatibility.js

```diff
@@ -41,8 +41,32 @@ function showBrowserCompatibilityWarning() {
     throw new Error('Browser does not suport ECMAScript 2017 / ES2017');
 }
 
+function isWebAssemblyEnable(){
+    if(!window.hasOwnProperty('WebAssembly') || typeof window.WebAssembly.instantiate !== "function") {
+        console.error('WebAssembly not supported');
+        return false;
+    }
+}
+
+function showWebAssemblyCompatibilityWarning() {
+
+    var imgpath   = OC.filePath('passwords', 'img', 'warning.png');
+        container = document.getElementById('main');
+    container.innerHTML =
+        '<div class="passwords-browser-compatibility passwords-jit-compatibility">' +
+        '<div class="message"><img class="warning-icon" src="' + imgpath +'"/>'+OC.L10N.translate('passwords', 'To view this website properly, please enable Javascript JIT.') +
+        '</div><div class="info" >' +
+        '<h3 class="howto">' + OC.L10N.translate('passwords', 'How to enable it') + '</h3>' +
+        '<p>' + OC.L10N.translate('passwords', 'JavaScript JIT might be disabled in your browser in order to render web content in a more secure configuration. You can always enable Javascript JIT in the settings, depending on your web browser.') + '</p><br />' +
```

Comment:
This text is placed directly onto the background image. This could be unreadable.
Its also very verbose. Additionally, neither Chromium nor Firefox do have a setting for this, both put it into the config / flags page. Additionally, calling the feature JIT may be cause them to misconfigure things in FF as typing JIT into the search of about:config will not show the actual setting to disable WASM. 
I feel like it would be better to keep the message short and just tell users that "This app requires WASM/WebAssembly to be enabled. Please check our Wiki to learn how to enable it" and then create a page in the wiki like `/Enable-WebAssembly`. This would also have the benefit that admins can provide a custom help page instead of users doing things they find on Google